bleak/backends/corebluetooth/client.py

```diff
@@ -381,3 +379,28 @@ async def stop_notify(
             raise BleakError(
                 "Could not stop notify on {0}: {1}".format(characteristic.uuid, success)
             )
+
+    async def update_RSSI(self, rtn: bool = False) -> int:
+        """To update RSSI value in dBm of the connected Peripheral
+
+        Args:
+            rtn (Bool): Whether to return the updated RSSI value or not
+
+        """
+        self._device_info.readRSSI()
+        manager = self._device_info.manager().delegate()
+        RSSI = manager.connected_peripheral.RSSI()
+        for i in range(20):  # First time takes a little otherwise returns None
+            RSSI = manager.connected_peripheral.RSSI()
+            if not RSSI:
+                await as_sleep(0.1)
```

Comment:
Import `asyncio` and use `asyncio.sleep` instead. That makes it adhere to how the rest of Bleak looks like.